src/makedeb/main.sh

```diff
@@ -105,6 +105,16 @@ generate_prefix_fields
 # Set pkgbase
 pkgbase="${pkgbase:-${pkgname[0]}}"
 
+if [[ "${pkgbase}" =~ [A-Z] ]]; then
+  error "'pkgbase' contains capital letters"
+  exit 1
+fi
+
+if [[ "${pkgname}" =~ [A-Z] ]]; then
+  error "'pkgname' contains capital letters"
+  exit 1
+fi
+
```

Comment:
This should be placed in `src/makepkg/functions/lint_pkgbuild/pkgname.sh`, as well any future linting checks.

I think there's some still in makedeb's code, but I'm wanting them to stay in makepkg if at all possible.